mreg_cli/utilities/api.py

```diff
@@ -394,7 +395,7 @@ def _check_expect_one_result(
     get_params = params.copy()
     # get_params["page_size"] = 1
     resp = get(path, get_params).json()
-    if "count" in resp and resp["count"] > max_hits_to_allow:
+    if max_hits_to_allow and resp.get("count", 0) > abs(max_hits_to_allow):
```

Comment:
Required in order to fetch _all_ networks in `network create`.

Reply:
Yeah, as much as I don't like default of allow all. Anyway, this should also be used to resolve https://github.com/unioslo/mreg-cli/blob/cd156d189ebcc4bae7f6a6986912f6796273c60b/mreg_cli/commands/permission.py#L55-L57 

(Pardon my hackery.)

Reply:
> Yeah, as much as I don't like default of allow all. Anyway, this should also be used to resolve

The default for `max_hits_to_allow` is still 500, it's just able to handle `None` arguments now to disable the limit where needed.

Reply:
Ah, right, explicit None, that makes a lot of sense. Thanks!

Reply:
> Yeah, as much as I don't like default of allow all. Anyway, this should also be used to resolve
> 
> https://github.com/unioslo/mreg-cli/blob/cd156d189ebcc4bae7f6a6986912f6796273c60b/mreg_cli/commands/permission.py#L55-L57
> 
> (Pardon my hackery.)

Fixed in [1d44a23](https://github.com/unioslo/mreg-cli/pull/238/commits/1d44a2325e2b7e6e0c9c081a5fe9e39d5d927007)